pragma-node/src/utils/sql.rs

```diff
@@ -34,8 +17,12 @@ pub const fn get_interval_specifier(
 ) -> Result<&'static str, InfraError> {
     if is_twap {
         match interval {
```

Comment:
This is no longer valid I think

Reply:
it is cause we still have some intervals not supported for twap